common-moonmage.lic

```diff
@@ -123,15 +123,18 @@ module DRCMM
 
   # There are many variants of a summoned moon weapon (blade, staff, sword, etc)
   # This function checks if you're holding one then tries to wear it.
+  # Returns true if what is in your hands is a summoned moon weapon that becomes worn.
+  # Returns false if you're not holding a moon weapon, or you are but can't wear it.
   # https://elanthipedia.play.net/Shape_Moonblade
   def wear_moon_weapon
```

Comment:
Can you also rename this to `wear_moon_weapon?` since its now a predicate?

Reply:
Will do. This change to return a boolean supports another spot I found in combat-trainer that was still doing the old “wear moon” action and script would hang because my character was wearing “trio of moons”.

https://github.com/rpherbig/dr-scripts/pull/4442/files#diff-772f69cb948ff35829a81342b2805fa8R839